compiler/src/iree/compiler/InputConversion/Common/ImportMLProgram.cpp

```diff
@@ -77,9 +82,86 @@ class MLProgramGlobalOpPattern
         srcOpAttr.has_value()
             ? Optional<TypedAttr>(srcOpAttr.value().cast<TypedAttr>())
             : llvm::None;
+    const bool isMutable = srcOp.getIsMutable();
+    const SymbolTable::Visibility visibility = srcOp.getVisibility();
     auto globalOp = rewriter.replaceOpWithNewOp<IREE::Util::GlobalOp>(
         srcOp, srcOp.getName(), srcOp.getIsMutable(), newType, srcOpTypedAttr);
-    globalOp.setVisibility(srcOp.getVisibility());
+    globalOp.setVisibility(visibility);
```

Comment:
we'll want to make these globals private once we expose the public accessors

Reply:
I flipped this to private only when accessors were created, let me know if I should just do that without checking (I think public isn't supported, so doing it unconditionally may be less surprising).